releng/tools.vitruv.dsls.parent/pom.xml

```diff
@@ -13,7 +13,8 @@
 	<packaging>pom</packaging>
 
 	<properties>
-		<vitruv.framework.url>https://vitruv-tools.github.io/updatesite/nightly/framework/</vitruv.framework.url>
+		<!-- A local updatesite for the framework can be specified by overwriting these properties. It defaults to the Vitruv updatesite. -->
+		<vitruv.framework.url>https://vitruv-tools.github.io/updatesite/nightly/</vitruv.framework.url>
```

Comment:
Actually, the reason is that the aggregated updatesite will only be available at `https://vitruv-tools.github.io/updatesite/nightly/`. The individual updatesites (`framework` etc.) will only contain the artifacts of the corresponding repository but not the dependencies anymore, such that we always need to reference the aggregated updatesite to resolve the dependencies. What is wrong, however, is that in case we overwrite the update path with a local one, the aggregated updatesite with the dependencies is not available anymore. I will provide a further commit that corrects that.